src/Resources/config/services.yml

```diff
@@ -92,6 +92,17 @@ services:
         class: Contao\CoreBundle\Security\User\ContaoUserProvider
         public: false
 
+    contao.store.session:
```

Comment:
Yeah but I'm with @aschempp here. Someday we will have to relocate this logic into a listener and he's not completely wrong when he says that it might also go into that listener so it would be responsible for both, storing as well as reloading those values for the current user.